sympy/polys/monomials.py

```diff
@@ -14,37 +14,36 @@
 @public
 def itermonomials(variables, max_degrees, min_degrees=None):
     r"""
-    `max_degrees` and `min_degrees` are either both integers or both lists.
-    Unless otherwise specified, `min_degrees` is either 0 or [0,...,0].
+    ``max_degrees`` and ``min_degrees`` are either both integers or both lists.
+    Unless otherwise specified, ``min_degrees`` is either ``0`` or
+    ``[0, ..., 0]``.
 
-    A generator of all monomials `monom` is returned, such that
+    A generator of all monomials ``monom`` is returned, such that
     either
-    min_degree <= total_degree(monom) <= max_degree,
+    ``min_degree <= total_degree(monom) <= max_degree``,
     or
-    min_degrees[i] <= degree_list(monom)[i] <= max_degrees[i], for all i.
+    ``min_degrees[i] <= degree_list(monom)[i] <= max_degrees[i]``,
+    for all ``i``.
 
-    Case I:: `max_degrees` and `min_degrees` are both integers.
-    ===========================================================
-    Given a set of variables `V` and a min_degree `N` and a max_degree `M`
-    generate a set of monomials of degree less than or equal to `N` and greater
-    than or equal to `M`. The total number of monomials in commutative
-    variables is huge and is given by the following formula if `M = 0`:
+    Case I. ``max_degrees`` and ``min_degrees`` are both integers
+    =============================================================
 
-        .. math::
+    Given a set of variables $V$ and a min_degree $N$ and a max_degree $M$
+    generate a set of monomials of degree less than or equal to $N$ and greater
+    than or equal to $M$. The total number of monomials in commutative
+    variables is huge and is given by the following formula if $M = 0$:
 
+        .. math::
             \frac{(\#V + N)!}{\#V! N!}
```

Comment:
Does `#V` mean the size of the set `V`? Maybe `|V|` would be better or perhaps just call it a "set of size V".